README.md

```diff
@@ -149,15 +146,12 @@ per time point:
 
 ## Solvers
 
-TPS uses by default a custom solver (`DataflowSolver`), but it also includes
-two symbolic solvers (`NaiveSymbolicSolver` and `BilateralSolver`) that
-implement the same functionality as the custom solver.
+TPS uses by default a custom solver (`DataflowSolver`). Historically, it also
+supported two symbolic solvers (`NaiveSymbolicSolver` and `BilateralSolver`)
+that implement the same functionality as the custom solver.
 
-We recommend using the default solver, which is the most recent and fastest of
-all three. Meanwhile, if you would like to use either of the two symbolic
-solvers on OS X, you will need to replace the `scalaz3.jar` with a packaged
```

Comment:
Good point, I will revert that change. I was initially trying to get the CircleCI integration tests to work with a newer version of ScalaZ3 compiled against Scala 2.12, but it didn't help. As you indicated above, I think removing this support is a good trade-off, especially since there is no feature regression if we proceed with this.